src/Sulu/Bundle/CustomUrlBundle/Resources/config/routing.xml

```diff
@@ -0,0 +1,125 @@
+        </service>
+        <service id="sulu_custom_urls.routing.route_enhancers.external_link"
+                 class="Sulu\Component\CustomUrl\Routing\Enhancers\ExternalLinkEnhancer">
+            <tag name="sulu.context" context="website"/>
+        </service>
+        <service id="sulu_custom_urls.routing.route_enhancers.seo"
+                 class="Sulu\Component\CustomUrl\Routing\Enhancers\SeoEnhancer">
+            <argument type="service" id="sulu_core.webspace.webspace_manager"/>
+            <tag name="sulu.context" context="website"/>
+        </service>
+
+        <service id="sulu_custom_urls.routing.router" class="Symfony\Cmf\Bundle\RoutingBundle\Routing\DynamicRouter">
+            <argument type="service" id="router.request_context"/>
+            <argument type="service" id="sulu_custom_urls.routing.nested_matcher"/>
+            <argument type="service" id="sulu_custom_urls.routing.generator"/>
+            <argument>%sulu_custom_urls.uri_filter_regexp%</argument>
+            <argument type="service" id="event_dispatcher" on-invalid="ignore"/>
+            <argument type="service" id="sulu_custom_urls.routing.provider"/>
+            <call method="setRequest">
+                <argument type="service" id="request" on-invalid="null" strict="false"/>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.trailing_slash"/>
+                <argument>100</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.trailing_html"/>
+                <argument>90</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.redirect"/>
+                <argument>80</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.seo"/>
+                <argument>70</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.content"/>
+                <argument>60</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.internal_link"/>
+                <argument>50</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.structure"/>
+                <argument>40</argument>
+            </call>
+            <call method="addRouteEnhancer">
+                <argument type="service" id="sulu_custom_urls.routing.route_enhancers.external_link"/>
+                <argument>30</argument>
+            </call>
```

Comment:
Why don't you do that as described in the [Symfony CMF documentation](http://symfony.com/doc/current/cmf/components/routing/dynamic.html#route-enhancer-compiler-pass)?